pkgs/servers/gotosocial/default.nix

```diff
@@ -9,11 +9,11 @@ let
   owner = "superseriousbusiness";
   repo = "gotosocial";
 
-  version = "0.17.0";
+  version = "0.17.1";
```

Comment:
Did you mean for this to be 0.17.2?

Reply:
what.. did i forget to git add before amending? I'll check and fix it in the morning..

Reply:
it's fixed now, sorry about the mess ^^